hack/istio/install-sleep-demo.sh

```diff
@@ -24,6 +24,9 @@ DELETE_SLEEP="false"
 while [[ $# -gt 0 ]]; do
   key="$1"
   case $key in
+    -a|--arch)
```

Comment:
This argument doesn't do anything at the moment, but it will be needed sooner or later, because there was a requirement to update sleep app for non-amd64 architectures. This seemed like the easiest solution, rather than messing with the hooks.ts file and than reverting to original state once the mentioned requirement is fulfilled